tests/snuba/tasks/test_unmerge.py

```diff
@@ -607,3 +684,111 @@ def collect_by_environment(aggregate, event):
         )
         assert destination_similar_items[1][0] == source.id
         assert destination_similar_items[1][1]["message:message:character-shingles"] < 1.0
+
+    @with_feature("organizations:escalating-issues-v2")
+    def test_no_dest(self):
```

Comment:
this is the only test that is relevant, the rest of it would just take too long to remove